import main

main.glogger.error("Crazy")